src/test/java/com/atlassian/bitbucket/jenkins/internal/fixture/BitbucketMockJenkinsRule.java

```diff
@@ -109,8 +110,8 @@ private void setupCredentials(String credentialId, String secret) throws Excepti
         CredentialsStore store = CredentialsProvider.lookupStores(jenkins).iterator().next();
         Domain domain = Domain.global();
         Credentials credentials =
-                new UsernamePasswordCredentialsImpl(
-                        CredentialsScope.GLOBAL, credentialId, "", "admin", secret);
+                new BitbucketTokenCredentialsImpl(
```

Comment:
There is probably going to be a merge conflict from this when my PR merges (or on my PR if this merges first)